pkgs/development/python-modules/crytic-compile/default.nix

```diff
@@ -2,10 +2,14 @@
 
 buildPythonPackage rec {
   pname = "crytic-compile";
-  version = "0.1.9";
+  version = "0.1.12";
 
   disabled = pythonOlder "3.6";
 
+  patchPhase = ''
+    substituteInPlace setup.py --replace "0.1.9" "0.1.12"
```

Comment:
do you mind extending this so that it include the package, I don't want to inadvertently also replace an unintended dependency.

Reply:
sure, forgot to change that after testing